### DISCUSSION TITLE
指南

### DISCUSSION THREAD

**Discussion by zhanyeye** [zhanyeye/clash-rules-lite#2] [General]
![image](https://user-images.githubusercontent.com/35565811/147398760-17324346-2fa3-4390-ad80-3d830ec8c58d.png)
![image](https://user-images.githubusercontent.com/35565811/147398721-88a75d2b-ce4d-4605-80a1-60871907f64d.png)

**Comment by katcu** [comment]
每次更新订阅都会覆盖路由规则吗？
